spring-boot-admin-server-ui/package.json

```diff
@@ -36,7 +36,8 @@
     "rxjs": "^6.3.3",
     "vue": "^2.5.17",
     "vue-clickaway": "2.2.1",
-    "vue-router": "^3.0.2"
+    "vue-router": "^3.0.2",
+    "vuejs-dialog": "^1.3.0"
```

Comment:
Please don't add this new dependency on `vuejs-dialog` either use our `sba-confirm-button` or implment the dialog.